va/va.h

```diff
@@ -1916,6 +2007,62 @@ typedef struct _VAContextParameterUpdateBuffer
     uint32_t reserved[VA_PADDING_MEDIUM];
 } VAContextParameterUpdateBuffer;
 
+/** \brief structure for encrypted segment info. */
+typedef struct _VAEncryptionSegmentInfo {
+  /** \brief  The offset relative to the start of the bitstream input in
+   *  bytes of the start of the segment*/
+  uint32_t segment_start_offset;
+  /** \brief  The length of the segments in bytes*/
+  uint32_t segment_length;
+  /** \brief  The length in bytes of the remainder of an incomplete block
+   *  from a previous segment*/
+  uint32_t partial_aes_block_size;
+  /** \brief  The length in bytes of the initial clear data */
+  uint32_t init_byte_length;
+  /** \brief  This will be AES 128 counter for secure decode and secure
```

Comment:
Good question. 
For near foreseeable future, HWDRM video streaming industry such as Google Play, Netflix, Disney etc. think that AES 128 bit encryption is sufficient so generalization would be waste but if we add generalization then this will be good design practice.

Reply:
is it possible ? if yes, I prefer to add generalization

Reply:
Sure. Ryan could you please add generalization?

Reply:
Updated